models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql

```diff
@@ -0,0 +1,209 @@
+{{ config (tags=['dunesql'],
+    schema='tokemak_ethereum',
+    alias = alias('view_lookup_tokens'),
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+        "project", 
+            "Tokemak",
+             \'["addmorebass"]\') }}'
+) }}
+
+
+WITH tokemak_ethereum_view_lookup_tokens(symbol, display_name, address, pricing_contract, decimals, is_pool, is_active, is_liability, is_dollar_stable) AS
```

Comment:
Hi @addmorebass,

I have transform tokemak_ethereum_view_tokemak_lookup_reactors.sql into `select ... from (values ...)` format:
https://github.com/duneanalytics/spellbook/blob/3456338155e4b868a00c1fe244301b153aea08fe/models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql

You can follow this format to transform this spell as well.